sound/soc/sof/debug-dsp-ops.c

```diff
@@ -8,6 +8,33 @@
 #include "sof-priv.h"
 #include "ops.h"
 
+static int sof_dsp_ops_boot_firmware(struct snd_sof_dev *sdev)
+{
+	const char *fw_filename;
+	int ret;
+
+	sdev->first_boot = true;
+
+	if (!sdev->test_profile.fw_path || !sdev->test_profile.fw_name) {
```

Comment:
I dont want to use the normal FW profile used on the DUT for regular audio functionality. They should be read only so that the user cannot modify it randomly